packages/block-renderer/src/components/block-renderer-container.tsx

```diff
@@ -88,19 +88,22 @@ const ScaledBlockRendererContainer = ( {
 	}, [] );
 
 	const scale = containerWidth / viewportWidth;
+	const maxHeightFor100vh = Math.max(
```

Comment:
I vote for the name `heightFor100vh` [as you said](https://github.com/Automattic/wp-calypso/pull/76132/files#r1175189390) 👍 
```suggestion
	const heightFor100vh = Math.max(
```